tdrs-backend/tdpservice/users/api/login.py

```diff
@@ -378,11 +382,13 @@ def get_auth_options(self, access_token, sub):
             userinfo_response = requests.post(ams_configuration["userinfo_endpoint"],
                                               {"access_token": access_token})
             user_info = userinfo_response.json()
-            logger.info(user_info)
-            # TODO Use `hhs_id` as primary authentication key
-            # See https://github.com/raft-tech/TANF-app/issues/1136#issuecomment-996822564
+            logging.debug(user_info)
+
             auth_options["username"] = user_info["email"]
-            if "hhs_id" in user_info:
-                auth_options["hhs_id"] = user_info.get("hss_id")
+            try:
+                auth_options["hhs_id"] = user_info["hhsid"]
```

Comment:
Very important to note the distinction left v. right as AMS provides a `hhsid` value and our code will continue to use `hhs_id`.